modular_skyrat/modules/milker/code/milker.dm

```diff
@@ -0,0 +1,305 @@
+#define VOLUME_LIGHT 1 // Volume 'light' mode extracts per action
+#define VOLUME_STRONG 5 // Volume 'strong' mode extracts per action
+#define HANDLE_LIGHT 1 // 'light' mode
+#define HANDLE_STRONG 2 // 'strong' mode
+#define HANDLE_MODE_LOWEST 1 // The lowest numbered setting currently available (so we dont need yet another list)
+#define HANDLE_MODE_HIGHEST 2 // The highest numbered setting currently available
+#define PLEASURE_LIGHT 1 // How much pleasure is gained through 'light' mode
+#define PLEASURE_STRONG 3 // How much pleasure is gained through 'strong' mode
+#define AROUSAL_LIGHT 1 // How much arousal is gained through 'light' mode
+#define AROUSAL_STRONG 3 // How much arousal is gained through 'strong' mode
+#define PAIN_LIGHT 3 // How much pain is gained through 'light' mode
+#define PAIN_STRONG 10 // How much pain is gained through 'strong' mode
+// Self-(and otherwise) Expression (of mammary organs)
+
+/// The expression device
+/obj/item/milker
+	name = "milker"
+	desc = "You and me, baby, ain't nothin' but mammals."
+	icon_state = "latexballon"
+	inhand_icon_state = "nothing"
+	force = 0
+	throwforce = 0
+	item_flags = DROPDEL | ABSTRACT | HAND_ITEM
+	attack_verb_continuous = list("slaps")
+	attack_verb_simple = list("slap")
+	hitsound = 'sound/effects/snap.ogg'
+	var/pleasure_amt = PLEASURE_LIGHT
+	var/arousal_amt = AROUSAL_LIGHT
+	var/pain_amt = PAIN_LIGHT // *chomp*
```

Comment:
You can write `amount`, two extra letters for readability won't kill you. Also, documentation :)

Reply:
Done and done.